src/HealthGPS.Core/interval.h

```diff
@@ -33,13 +41,7 @@ template <Numerical TYPE> class Interval {
     /// @brief Determines whether a value is in the Interval.
     /// @param value The value to check
     /// @return true if the value is in the interval; otherwise, false.
-    bool contains(TYPE value) const noexcept {
-        if (lower_ < upper_) {
-            return lower_ <= value && value <= upper_;
-        }
-
-        return lower_ >= value && value >= upper_;
-    }
+    bool contains(TYPE value) const noexcept { return lower_ <= value && value <= upper_; }
```

Comment:
Do we want to change this behaviour? I agree that your way is saner, but I'm a little concerned that something might rely on this.

Reply:
I did test the existing model, and it didn't trip the exception. I can't see anywhere that relies on non-sane intervals.